code/game/objects/items/weapons/clown_items.dm

```diff
@@ -121,12 +121,13 @@
 
 /obj/item/weapon/bikehorn/attack_self(mob/user)
 	if(!spam_flag)
-		spam_flag = 1
+		spam_flag = TRUE

```

Comment:
```TRUE``` is actually just ```#define TRUE 1``` inside stdlb.dm, so there's literally no change other than better readability.